.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.7
```

Comment:
So is this still necessary now that it's defined dynamically in the Gemfile? I don't understand who exactly consumes that file.